lld/MachO/Driver.cpp

```diff
@@ -1322,6 +1323,38 @@ static void gatherInputSections() {
   }
 }
 
+static void codegenDataGenerate() {
+  TimeTraceScope timeScope("Generating codegen data");
+
+  OutlinedHashTreeRecord globalOutlineRecord;
+  for (ConcatInputSection *isec : inputSections) {
```

Comment:
nit: You can remove the braces in the for loop. Though in practice, I've seen both conventions be used here. So up to you if you think that helps with readability.